Source/OxyPlot.Wpf.Shared/PlotViewBase.cs

```diff
@@ -126,7 +126,7 @@ protected PlotViewBase()
         /// <summary>
         /// Gets the coordinates of the client area of the view.
         /// </summary>
-        public OxyRect ClientArea => new OxyRect(0, 0, this.ActualWidth, this.ActualHeight);
+        public OxyRect ClientArea => new(0, 0, this.ActualWidth, this.ActualHeight);
```

Comment:
I don't like this

Reply:
For me, the upside of forcing `var` would be mainly that there is a clear rule that is easy to remember and follow. Leaving it up to the individual will lead to some parts of the code base where `var` is used extensively, and others where it isn't. Of course really this isn't a problem at all, just slightly triggers my obsessive-compulsive tendencies ;)

I'd be fine with leaving `var` and target-typed new unspecified for now. I'm not quite sure if I can revert just the changes to `var` and `new` and still keep the rest of the refactorings... In the worst case I'll have to revert everything and start from the beginning. I'll have to look into this, probably won't manage before next week.

Reply:
Sorry, did not really find time to continue here. Also lacking a bit of motivation, as it does seem that I pretty much would have to start over - I did not find an easy way to revert just the changes relating to `var`.

I will be away for a couple of weeks starting this weekend, and probably will not be able to do anything during this time. If you, or someone else would like to pick this up this is perfectly fine by me, otherwise I will take another look in ~4 weeks.